luigi/lock.py

```diff
@@ -33,9 +33,12 @@ def getpcmd(pid):
 
     :param pid:
     """
-    cmd = 'ps -p %s -o command=' % (pid,)
+    cmd = 'ps -o pid,comm | grep %s | xargs -n 2' % (pid,)
```

Comment:
How does this work? Can you use pipes in Popen? Pipes is a feature of the shell right?

EIther way probably easier to parse it in Python